x/scheduler/keeper/keeper_setup_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	tmdb "github.com/tendermint/tm-db"
 )
 
+//nolint:deadcode
 func newSchedulerKeeper(t testing.TB) (*Keeper, sdk.Context) {
```

Comment:
We can remove the dead code. Let's definitely clean this up.

Reply:
Done in the commit https://github.com/palomachain/paloma/pull/755/commits/0d65fc0c9f5a307ac368632d2d9043624b547834